neuraxle/steps/numpy.py

```diff
@@ -92,6 +114,25 @@ def __init__(self):
         BaseStep.__init__(self)
         NonFittableMixin.__init__(self)
 
+    def join_data_containers(
```

Comment:
Same comment here, except here you don't inherit from NumpyConcatenateOnCustomAxis so it's different.